src/ru/fizteh/fivt/students/hromov_igor/multifilemap/base/DBaseTable.java

```diff
@@ -0,0 +1,196 @@
+package ru.fizteh.fivt.students.hromov_igor.multifilemap.base;
+
+import ru.fizteh.fivt.storage.strings.Table;
+import ru.fizteh.fivt.students.hromov_igor.multifilemap.base.exception.*;
+
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.Map.Entry;
+
+class IntPair {
```

Comment:
IntPair -> DirectoryAndFileDescriptor
x -> directoryIndex
y -> fileIndex

"I hate pair in C++ and I would recommend not to implement and not to
use such a class in Java. The class name pair says absolutely nothing
about what it is really good for and what is the actual relation of
the data in a pair. It just expresses a minor property of the data,
that it split into two parts. But it is still just data. So with a
pair class in your program you have data hanging around in the program
screaming "look at me, I am data, data consisting of two parts".

My answer to that is: Who cares? Objects should have meaningful
operations. Not just "store something and you can get it back". Their
classes should have meaningful names. Map.Entry() is such a meaningful
name, since it indicates what the data is about. Pair() means nothing.
Two farts in the same direction."